automatically provide links and close issues that depend on fixes in …

titusfortner opened the pull request:
I've seen us send people to other projects regularly. How about if we create specific labels for these so we can track them, as well as make sure we are providing all the necessary info to users?

Any suggestions on content to make this more clear?

diemol commented on .github/label-commenter-config.yml:
```diff
@@ -31,6 +31,44 @@ labels:
+          This issue has been determined to require a change to the
+          [WebDriver W3C Specification](https://w3c.github.io/webdriver/) for Selenium to be able to support it.
+
+          Please [Create an Issue](https://github.com/w3c/webdriver/issues/new) with the webdriver project.
```
```suggestion
          Please [create an issue](https://github.com/w3c/webdriver/issues/new) with the WebDriver project.
```

diemol commented on .github/label-commenter-config.yml:
```diff
@@ -31,6 +31,44 @@ labels:
+      issue:
+        body: |
+          Hi, @{{ issue.user.login }}.
+          This issue has been determined to require fixes in [chromedriver](https://chromedriver.chromium.org/home).
```
```suggestion
          This issue has been determined to require fixes in [ChromeDriver](https://chromedriver.chromium.org/home).
```

diemol commented on .github/label-commenter-config.yml:
```diff
@@ -31,6 +31,44 @@ labels:
+
+          You can see if the feature is passing in the [Web Platform Tests](https://wpt.fyi/results/webdriver/tests).
+
+          If it is something new, please [Create an Issue](https://bugs.chromium.org/p/chromedriver/issues/list) with the chromedriver team.
```
```suggestion
          If it is something new, please [create an issue](https://bugs.chromium.org/p/chromedriver/issues/list) with the ChromeDriver team.
```

diemol commented on .github/label-commenter-config.yml:
```diff
@@ -31,6 +31,44 @@ labels:
+      issue:
+        body: |
+          Hi, @{{ issue.user.login }}.
+          This issue has been determined to require fixes in [geckodriver](https://firefox-source-docs.mozilla.org/testing/geckodriver/).
```
```suggestion
          This issue has been determined to require fixes in [GeckoDriver](https://firefox-source-docs.mozilla.org/testing/geckodriver/).
```

diemol commented on .github/label-commenter-config.yml:
```diff
@@ -31,6 +31,44 @@ labels:
+
+          You can see if the feature is passing in the [Web Platform Tests](https://wpt.fyi/results/webdriver/tests).
+
+          If it is something new, please [Create an Issue](https://github.com/mozilla/geckodriver/issues/new) with the geckodriver team.
```
```suggestion
          If it is something new, please [Create an Issue](https://github.com/mozilla/geckodriver/issues/new) with the GeckoDriver team.
```

diemol reviewed (changes requested):
Thank you, @titusfortner!

Maybe we can add a couple comments [here](https://github.com/SeleniumHQ/selenium/blob/23c6171614bfd5e927acac1e68eb0099a42b558d/.github/label-commenter-config.yml#L17-L27), telling maintainers about the presence of these new labels?

Also, do the labels exist already?

titusfortner commented:
I added a note to maintainers and added the applicable labels.

I think the main question left is whether we want these to auto-close the issues, since G-w3c does not do that currently, so it would be a change. @AutomatedTester what's your take?

diemol commented on .github/label-commenter-config.yml:
```diff
@@ -23,6 +23,10 @@ labels:
               <p>
                 If the issue is valid but there is no time to troubleshoot it, consider adding the <code>help wanted</code> label.
               </p>
+              <p>
+                If the issue requires changes or fixes from an external project (e.g., chromedriver, geckodriver, w3c),
+          add the applicable <code>G-*</code> label, and it will provide the correct link and auto-close the issue.
+              </p>
```
```suggestion
              <p>
                If the issue requires changes or fixes from an external project (e.g., ChromeDriver, GeckoDriver, W3C),
               add the applicable <code>G-*</code> label, and it will provide the correct link and auto-close the 
               issue.
              </p>
```

diemol commented:
Let's merge as it is. We can always change things afterwards.
